prow/plugins.yaml

```diff
@@ -316,6 +317,7 @@ plugins:
 
   kubernetes/sig-release:
   - approve
+  - stage
```

Comment:
added for sig-release as well, as per https://github.com/kubernetes/test-infra/pull/8335#issuecomment-396720161